### PR TITLE
BZ1680557 Registry updates

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1680557

### CODE REVIEW

File: modules/registry-securing-manually.adoc [hunk 1]
```diff
@@ -13,11 +13,11 @@ sections for steps on how to manually do so.
 Manually secure the registry to serve traffic via TLS:
 
 ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
-. Deploy the registry using the `oc adm registry` command as a
+. Deploy the registry using the `oc registry` command as a
 user with cluster administrator privileges, for example:
 +
 ----
-$ oc adm registry --config=/etc/origin/master/admin.kubeconfig \//<1>
+$ oc registry --config=/etc/origin/master/admin.kubeconfig \//<1>
```
Comment (wzheng1) [inline modules/registry-securing-manually.adoc]: I don't think command 'oc registry' can deploy registry manually. You can see its function is as below:
Advanced commands:
  info        Print info about the integrated registry
  login       Login to the integrated registry

File: modules/registry-viewing-contents.adoc [hunk 2]
```diff
@@ -42,7 +42,7 @@ the container image registry:
 . List the registry contents using the `oc rsh` command:
 +
 ----
-# oc rsh dc/podman-registry find /registry
+# oc rsh podman-registry find /registry
```
Comment (wzheng1) [inline modules/registry-viewing-contents.adoc]: In 4.0, only S3 storage is supported, there will be no content under registry folder. Plus, no need to ssh into node to access registry pod, user can use kube:admin account to login and then check the pods under project openshift-image-registry like below:
$ oc get pods
NAME                                               READY   STATUS    RESTARTS   AGE
cluster-image-registry-operator-764bd7f846-qqtpb   1/1     Running   0          78m
image-registry-79fb4469f6-llrln                    1/1     Running   0          77m
node-ca-hjksc                                      1/1     Running   0          73m
node-ca-tftj6                                      1/1     Running   0          77m
node-ca-wb6ht                                      1/1     Running   0          77m
node-ca-zvt9q                                      1/1     Running   0          74m

File: modules/registry-viewing-logs.adoc [hunk 3]
```diff
@@ -13,7 +13,7 @@ You can view the logs for the registry by using the `oc logs` command.
 for the container image registry:
 +
 ----
-$ oc logs dc/podman-registry
+$ oc logs podman-registry
```
Comment (wzheng1) [inline modules/registry-viewing-logs.adoc]: It is better to change the pod name to the one like this 'image-registry-79fb4469f6-llrln'

**Comment by bmcelvee** [comment]
I'm going ahead and merging this PR so we can continue updating the registry documentation, thanks! :)

**Pull request merged**